src/ampycloud/data.py

```diff
@@ -992,6 +1010,28 @@ def layers(self) -> pd.DataFrame:
         identified by the layering algorithm. """
         return self._layers
 
+    @property
+    def high_clouds_detected(self) -> bool:
+        """ Returns whether high clouds were detected in the data.
+
+        Returns:
+            bool: whether high clouds were detected.
+
+        """
+        return self.data.attrs['high_clouds_detected']
+
+    def _ncd_or_nsc(self) -> str:
+        """ Return the METAR code for No Cloud Detected / No Significant Cloud.
+        Decision based on the attribute set in data.attrs.
+
```

Comment:
```suggestion

Note that regulation says that "if there are no clouds of operational significance and no restriction on vertical visibility and the abbreviation ‘CAVOK’ is not appropriate, the abbreviation ‘NSC’ should be used" (AMC1 MET.TR.205(e)(1)). Therefore, the NSC returned by ampycloud only covers the case of no clouds of operational significance (except CB and TCU). 

```

Reply:
I think we should also/only put this sentence (or similar) into the "Scope of ampycloud" section here: https://meteoswiss.github.io/ampycloud/index.html

Reply:
I would propose the following tweak to this proposition, to also include the limitations of the 'NCD'. I would agree to have this in the online docs, and I'll also add it to the scientific article.

```
Note that regulation says that "if there are no clouds of operational significance and no restriction on vertical visibility and the abbreviation ‘CAVOK’ is not appropriate, the abbreviation ‘NSC’ should be used" (AMC1 MET.TR.205(e)(1)). ampycloud cannot decide whether a 'CAVOK' is appropriate, and will therefore always return 'NSC' if no clouds of operational significance are found. If no clouds are detected at all by the ceilometers, ampycloud will return 'NCD'. Importantly, users should bear in mind that ampycloud cannot handle CB and TCU cases, such that any 'NCD'/'NSC' codes issued may need to be overwritten by the user in certain situations. 
```

Reply:
added it to `scope.rst`